Formula/juju.rb

```diff
@@ -1,9 +1,11 @@
 class Juju < Formula
   desc "DevOps management tool"
   homepage "https://jujucharms.com/"
+  # https://github.com/Homebrew/homebrew-core/pull/57456#issuecomment-656703975
   url "https://github.com/juju/juju.git",
-    :tag      => "juju-2.8.1",
-    :revision => "16439b3d1c528b7a0e019a16c2122ccfcf6aa41f"
+    :tag      => "juju-2.8.0",
```

Comment:
We should probably downgrade to `juju-2.7.7` as that is marked as the latest release.
https://github.com/juju/juju/tree/juju-2.7.7

cc @SimonRichardson

Reply:
We maintain several tracks of Juju and `2.8.0` came out before `2.7.7`, but `2.8.0` is in fact the latest one. I think this is just the way github picks the "latest" one (if you can pin the latest version, that would be good to learn)

So this PR is actually correct and `2.8.0` is the one we want.

Reply:
I believe you can pin `2.8.0` as the latest (by creating a new version). Homebrew has its own way to do the latest version check (`livecheck`), and many people use that as source of truth to do the voluntary version bump.

What I can think of:
- create a new latest release version (we can use it to track down that) (this is the [PR modification](https://github.com/Homebrew/homebrew-livecheck/pull/1085) for it)
- have the recommended version specified somewhere, we can use `livecheck` to track down that

Reply:
I will merge the downgrade action first, but the `livecheck` is still an issue (we still need to have certify the source of the truth)

Reply:
Sure I'll check what options we have available to us.

@chenrui333 Thanks for the info & help :+1:

Reply:
Sounds great, feel free to circle back on this thread.